backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/PodiumApi.java

```diff
@@ -0,0 +1,34 @@
+package org.molgenis.emx2.web;
+
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import org.jetbrains.annotations.NotNull;
+import org.molgenis.emx2.web.service.PodiumService;
+
+public class PodiumApi {
+  private static PodiumService podiumService = null;
+
+  private PodiumApi() {}
+
+  public static void create(Javalin app) {
+    final String reportPath = "/api/podium";
+    app.post(reportPath, PodiumApi::handlePodiumRequest);
+    podiumService = new PodiumService(HttpClient.newHttpClient());
+  }
+
+  private static void handlePodiumRequest(@NotNull Context context)
```

Comment:
Is it difficult to also test this method, to make sonar happy?

Reply:
I could just add random code to make it happy, but that would be silly 😅